straxen/plugins/acqmon_processing.py

```diff
@@ -216,3 +216,125 @@ def compute(self, events, aqmon_hits):
         result['time'] = events['time']
         result['endtime'] = events['endtime']
         return result
+
+
+DIGITIZER_DELAY_CHANGE_RUNID = '021286'
+TO_BE_EXCPECTED_MAX_DELAY = 11e3
```

Comment:
Also maybe for the sake of those who come after us, this isn't the correct spelling of 'expected'.